data/mods/Aftershock/items/gun/25mm.json

```diff
@@ -0,0 +1,48 @@
+[
+  {
+    "id": "afs_gibs_shotgun",
+    "looks_like": "m79",
+    "type": "GUN",
+    "reload_noise_volume": 10,
+    "name": { "str": "Gibson S86" },
+    "description": "The Gibson S86 is an extremely powerful 5-gauge combat shotgun manufactured by Gibson Armory, designed to take care of heavily armored opponents like power armor users.  However due to its recoil, the gun is only usable if its either mounted, or wielded by a person with sufficent strength like a power armor user.\n\nDue to its size and build, it feels more like a grenade launcher than an actual shotgun, and can actually chamber light grenade ammo.",
```

Comment:
Is it really a shotgun? My understanding was that it was a grenade launcher, and you can happen to load shot-equivalent shells into it.

Mostly say this because real 40mm grenade launchers have shells that are basically oversized 00 shot [the one that says canister in here ](https://en.wikipedia.org/wiki/40_mm_grenade). But having guns designed shotguns shoot grenades seems weirder

Reply:
Ah that makes sense; can re-write the flavor for this then!